### ISSUE TITLE
NTR: Bartonella henselae PCR positive in blood or tissue (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8067]
New term request Bartonella henselae PCR positive in blood or tissue (samant_101230221701): Polymerase chain reaction; DNA  UMLS: 2748143

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000550
- label: Positive Bartonella henselae nucleic acid test in the blood circulation
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: 
- def: Detection of nucleic acid of Bartonella henselae in the blood circulation by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8067
- synonyms:
